qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -126,7 +126,19 @@
             Windows:
                 # Windows guest only works with qxl driver.
                 vga = "qxl"
+        - check_hotplug_virtio_console:
+            only default
+            gagent_check_type = hotplug_virtio_console
+            extra_params += "-device virtio-serial-pci,id=virtio-serial0,bus=pci.0,addr=0x7"
+            char_id = "serial0"
+            char_path = "/tmp/org.qemu.guest_agent.0"
+            hotplug_gagent_serial_type = virtio
+            dev_driver = "virtserialport"
+            dev_id = "serial0"
+            dev_name = "org.qemu.guest_agent.0"
     variants:
+        - default:
```

Comment:
In my opinion, if variants 'virtio_serial' and 'isa_serial' can cover all situations, you don't need to add a new variant.

Reply:
Yes, but hotplug/unplug guest agent device without any additional serial in the case. So I add the default variant.

Reply:
can we have it as @default